docs/effects/api.md

```diff
@@ -93,6 +93,51 @@ export class SomeEffectsClass {
 }
 ```
 
+### Initializing effect
+You can execute some code that will be executed directly after the effect class is loaded.
+```ts
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { defer } from 'rxjs/observable/defer';
+
+@Injectable()
+export class SomeEffectsClass {
+  constructor(private actions$: Actions) { }
+
+  @Effect({ dispatch: false }) init$: Observable<any> = defer(() => {
+    return of(console.log('init$'));
+  });
+}
+```
+
+If you want to trigger another action, be careful to add this effect at the end.
+```ts
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { defer } from 'rxjs/observable/defer';
+import { LoginAction, LogoutAction } from './auth';
+
+@Injectable()
+export class SomeEffectsClass {
+  constructor(private actions$: Actions) { }
+  
+  @Effect() authActions$ = this.action$.pipe(
```

Comment:
Add a `{ dispatch: false }` here, or the effect will error if you copy/paste this code as-is.

Reply:
Fix the spacing in the code snippet to be 2 spaces.